fixtures_test.go

```diff
@@ -0,0 +1,30 @@
, -0.01757652, -0.046771515, 0.006021731, 0.0350991, -0.055127304, 0.0004067387, 0.0026264484, -0.00842346, -0.029608497, -0.043472905, -0.045593407, 0.030926447, -0.012382723, -0.039737374, -0.016628245, -0.028230408, 0.042542323, 0.024740454, 0.029905234, -0.021382011, 0.0087046465, 0.008666167, -0.01237462, 0.0058083804, 0.049879998, 0.027694289, -0.033726525, 0.018072732, 0.013075523, 0.04129341, -0.003759638, 0.025136042, -0.0048480816, -0.08633937, 0.026146557, 0.00095875195, -0.007239596, 0.030176798, 0.012911557, -0.05567729, 0.101863965, -0.012008359, -0.019885525, 0.06285349, -0.020306658, 0.021225194, 0.03904156, -0.018150982, 0.005765315, -0.014720191, 0.039356854, -0.05162945, -0.010809988, -0.019877022, -0.043858252, -0.016290769, -0.00052574894, 0.00071051397, -0.05228283, 0.00212076, 0.10330768, 0.03806923, -0.011989388, -0.0092820525, -0.07842139, -0.005238204, 0.0050059403, 0.004792647, -0.028316176, 0.008777532, -0.055501916, 0.0031685885, -0.044218272, -0.040338665, -0.035807323, 0.003954393, -0.059668504, 0.051771026, 0.023667619, 0.0073676254, -0.00062780967, -0.09302048, -0.028653411, 0.022346793, -0.0054721106, -0.035974275, 0.011861464, -0.044336487, -0.020843782, -0.034995276, 0.0073804525, -0.05347183, 0.027588347, 0.012416945, 0.038829893, 0.024596987, 0.0074599786, 0.014858577, 0.03851009, 0.014121627, -0.04871009, 0.027252283, 0.0553166, 0.05215012, 0.018370263, 0.0530953, 0.031772766, -0.017525164, -0.062617734, -0.008032683, 0.060756683, 0.037613112, -0.06970549, -0.03628826, 0.0060448144, 0.0106271915, 0.01570074, -0.032992955, 0.08146692, -0.0005743188, -0.003664223, 0.029400827, -0.007885519, -0.08933015, -0.019689526, 0.015134545, 0.0032099236, -0.02092406, -0.014055446, 0.010172088, 0.02639813, 0.028534463, 0.03089289, -0.017581366, -0.026170125, -0.028410664, -0.007110761, 0.052456588, 0.004216275, -0.0051908707, 0.02629775, 0.035634518, -0.024233747, -0.020206803, -0.08185167, -0.038406268, -0.019373732, 0.03597261, -0.063467175, -0.01582219, 0.05335854, -0.0067784563, 0.025629507, -0.005729798, 0.011919803, -0.049962502, 0.02882643, 0.040532112, 0.037180837, -0.0654546, -0.0078468155, 0.0056370874, -0.013275107, -0.012502071, -0.016623577, -0.00042084648, 0.0064675817, 0.05863257, 0.03738945, -0.021043384, 0.03962018, 0.011263337, -0.074983895, -0.026997007, 0.051713623, 0.019749667, 0.022187935, -0.02188655, -0.026837206, 0.0038193846, 0.019730948, 0.026408449, -0.073553264, 0.054254226, -0.021037942, -0.011475345, -0.008065046, -0.0041081496, 0.07153666, -0.0011454923, 0.0072494335, -0.02546474, -0.034410242, -0.008523845, -0.004515808, -0.0043068486, 0.06108401, 0.04297906, 0.04093538, 0.03570107, -0.032012034, 0.015930245, -0.043826263, 0.021157239, 0.029291723, -0.031764403, 0.032508258, -0.007936765, -0.0007478923, -0.05491598, -0.021232825, -0.011368038, 0.04037207, 0.028925741, 0.02602244, -0.022289654, 0.010383836, 0.0031105224, -0.0002327807, -0.04982281, -0.026878828, -0.026498057, -0.038631085},
```

Comment:
I'm getting different numbers for all prompts, but it seems to change with the Ollama version? I ran this, then updated Ollama and ran it again, and got different numbers both times (also different between the two runs). I didn't expect this, but maybe it's normal? Just to double check, can you run with the latest `v0.1.44` please? (There's a `v0.1.45` but it's pre-release).

---

On my side I'm running the code on an x86_64 Intel CPU, Fedora  39 (Kernel 6.8).

Ran with:

<details><summary>test code</summary>

```go
package chromem

import (
	"context"
	"fmt"
	"os"
	"testing"
)

func TestFoo(t *testing.T) {
	e := NewEmbeddingFuncOllama("nomic-embed-text", os.Getenv("OLLAMA_HOST"))

	ee := func(s string) []float32 {
		v, err := e(context.Background(), s)
		if err != nil {
			panic(err)
		}
		return v
	}

	testEmbeddings := map[string][]float32{
		"search_document: Village Builder Game": ee("search_document: Village Builder Game"),
		"search_document: Town Craft Idle Game": ee("search_document: Town Craft Idle Game"),
		"search_document: Some Idle Game":       ee("search_document: Some Idle Game"),
		"search_query: town":                    ee("search_query: town"),
		"search_query: idle":                    ee("search_query: idle"),
	}

	fmt.Printf("%#v\n", testEmbeddings)
}
```

</details>

And then: `go test -v -run TestFoo -count 1 .`

Reply:
Yeah embeddings were kinda broken in some versions of Ollama. See: https://github.com/ollama/ollama/issues/3777 and https://github.com/ollama/ollama/issues/3053. I used `v0.1.31` to generated them originally. I have re-generated them with `v0.1.45` now, which changes them slightly again 🤷‍♂️